### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/advanced/templates.md`

### PR DESCRIPTION
**Pull request by Heumhub**
Add Korean translation for docs/ko/docs/advanced/templates.md

### CODE REVIEW

File: docs/ko/docs/advanced/templates.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+**FastAPI**와 함께 원하는 어떤 템플릿 엔진도 사용할 수 있습니다.
+
+일반적인 선택은 Jinja2로, Flask와 다른 도구에서도 사용됩니다.
+
+설정을 쉽게 할 수 있는 유틸리티가 있으며, 이를 **FastAPI** 애플리케이션에서 직접 사용할 수 있습니다(Starlette 제공).
+
+## 의존성 설치
+
+가상 환경을 생성하고(virtual environment{.internal-link target=_blank}), 활성화한 후 jinja2를 설치해야 합니다:
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/templates.md:11]:
```suggestion
가상 환경을 생성하고(virtual environment{.internal-link target=_blank}), 활성화한 후 jinja2를 설치해야 합니다:
```
markdown has broken.
please check your work with generated docs by github action

File: docs/ko/docs/advanced/templates.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+</div>
+
+## 사용하기 `Jinja2Templates`
+
+* `Jinja2Templates`를 가져옵니다.
+* 나중에 재사용할 수 있는 `templates` 객체를 생성합니다.
+* 템플릿을 반환할 경로 작업에 `Request` 매개변수를 선언합니다.
+* 생성한 `templates`를 사용하여 `TemplateResponse`를 렌더링하고 반환합니다. 템플릿의 이름, 요청 객체 및 Jinja2 템플릿 내에서 사용될 키-값 쌍이 포함된 "컨텍스트" 사전도 전달합니다.
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/templates.md:29]:
```suggestion
* 생성한 `templates`를 사용하여 `TemplateResponse`를 렌더링하고 반환합니다. 템플릿의 이름, 요청 객체 및 Jinja2 템플릿 내에서 사용될 키-값 쌍이 포함된 "컨텍스트" 딕셔너리도 전달합니다.
```

File: docs/ko/docs/advanced/templates.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+
+```Python hl_lines="4  11  15-18"
+{!../../docs_src/templates/tutorial001.py!}
+```
+
+/// 노트
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/templates.md:36]:
```suggestion
/// 노트
```
markdown has broken here

File: docs/ko/docs/advanced/templates.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+`response_class=HTMLResponse`를 선언하면 문서 UI 응답이 HTML임을 알 수 있습니다.
+
+///
+
+/// 노트 | "기술 세부 사항"
+`from starlette.templating import Jinja2Templates`를 사용할 수도 있습니다.
+
+**FastAPI**는 개발자를 위한 편리함으로 `fastapi.templating` 대신 `starlette.templating`을 제공합니다. 하지만 대부분의 사용 가능한 응답은 Starlette에서 직접 옵니다. `Request` 및 `StaticFiles`도 마찬가지입니다.
+///
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/templates.md:54]:
```suggestion
///

/// tip | 참고

`response_class=HTMLResponse`를 선언하면 문서 UI 응답이 HTML임을 알 수 있습니다.

///

/// 노트 | "기술 세부 사항"
`from starlette.templating import Jinja2Templates`를 사용할 수도 있습니다.

**FastAPI**는 개발자를 위한 편리함으로 `fastapi.templating` 대신 `starlette.templating`을 제공합니다. 하지만 대부분의 사용 가능한 응답은 Starlette에서 직접 옵니다. `Request` 및 `StaticFiles`도 마찬가지입니다.
///
```
you need to your generated docs (2)

File: docs/ko/docs/advanced/templates.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+## 템플릿 작성하기
+
+그런 다음 `templates/item.html`에 다음과 같은 템플릿을 작성할 수 있습니다:
```
Comment (sodaMelon) [inline docs/ko/docs/advanced/templates.md:58]:
```suggestion
그런 다음 `templates/item.html`에 템플릿을 작성할 수 있습니다. 예를 들면:
```

File: docs/ko/docs/advanced/templates.md
Comment (kim-sangah) [inline docs/ko/docs/advanced/templates.md:1]: LGTM!

**Review by kim-sangah — APPROVED** [review]

File: docs/ko/docs/advanced/templates.md
Comment (Heumhub) [reply docs/ko/docs/advanced/templates.md:1]: thank you!

**Review by sodaMelon — APPROVED** [review]
LGTM!

File: docs/ko/docs/advanced/templates.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+* 나중에 재사용할 수 있는 `templates` 객체를 생성합니다.
+* 템플릿을 반환할 경로 작업에 `Request` 매개변수를 선언합니다.
+* 생성한 `templates`를 사용하여 `TemplateResponse`를 렌더링하고 반환합니다. 템플릿의 이름, 요청 객체 및 Jinja2 템플릿 내에서 사용될 키-값 쌍이 포함된 "컨텍스트" 딕셔너리도 전달합니다.
+
+
+```Python hl_lines="4  11  15-18"
+{!../../docs_src/templates/tutorial001.py!}
+```
+
+/// 노트
```
Comment (hard-coders) [inline docs/ko/docs/advanced/templates.md:36]:
```suggestion
/// note | 참고
```

File: docs/ko/docs/advanced/templates.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+
+FastAPI 0.108.0 이전과 Starlette 0.29.0에서는 `name`이 첫 번째 매개변수였습니다.
+
+또한 이전 버전에서는 `request` 객체가 Jinja2의 컨텍스트에서 키-값 쌍의 일부로 전달되었습니다.
+
+///
+
+/// tip | 참고
```
Comment (hard-coders) [inline docs/ko/docs/advanced/templates.md:44]:
```suggestion
/// tip | 팁
```

File: docs/ko/docs/advanced/templates.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+
+`response_class=HTMLResponse`를 선언하면 문서 UI 응답이 HTML임을 알 수 있습니다.
+
+///
+
+/// 노트 | "기술 세부 사항"
```
Comment (hard-coders) [inline docs/ko/docs/advanced/templates.md:50]:
```suggestion
/// note | "기술 세부 사항"
```

**Review by hard-coders — CHANGES_REQUESTED** [review]
It's broken. See [Translation specific tips and guidelines](https://fastapi.tiangolo.com/contributing/#translation-specific-tips-and-guidelines) and https://github.com/fastapi/fastapi/discussions/3167#discussioncomment-721448

> In lines starting with `///` translate only the text part after `|`. Leave the rest unchanged.

File: docs/ko/docs/advanced/templates.md (hunk 8 above)
Comment (Heumhub) [reply docs/ko/docs/advanced/templates.md:50]: 감사합니다

**Comment by alejsdev** [comment]
Thanks for your contribution! @Heumhub :partying_face: 
And thanks for the reviews @sodaMelon, @hard-coders and @kim-sangah :mag:

**Pull request merged**